app/Http/Controllers/API/EventController.php

```diff
@@ -88,6 +87,7 @@ public function getEventsByUsersNetworks(Request $request, $date_from = null, $d
                  'value' => $party->location,
                  'latitude' => $party->latitude,
                  'longitude' => $party->longitude,
+                 'area' => $group->area,
```

Comment:
Coming back as null - needs to be $group['area'] here I believe.

Reply:
Needs to be in both sections.  Try now?